tests/wicked/before_test.pm

```diff
@@ -105,7 +106,9 @@ sub run {
             }
         }
         if (check_var('WICKED', 'ipv6')) {
-            zypper_ar('http://download.suse.de/ibs/home:/wicked-maintainers:/openQA/SLE_15_SP2/', name => 'wicked_maintainers', no_gpg_check => 1, priority => 60);
+            my $repo_url='https://download.opensuse.org/repositories/home:/asmorodskyi/';
+            $repo_url='http://download.suse.de/ibs/home:/wicked-maintainers:/openQA/' if(is_sle());
+            zypper_ar($repo_url . generate_version('_') . '/', name => 'wicked_maintainers', no_gpg_check => 1, priority => 60);
```

Comment:
I would not use `name=>'wicked_maintainers'` as it is actually the alias. And I had problems with it.
```suggestion
            zypper_ar($repo_url . generate_version('_') . '/', params => '-n wicked_maintainers', no_gpg_check => 1, priority => 60);
```

Reply:
hm ok the error on my side was, that I specified a `*.repo` file with a alias... so forget about my comment